controller/cache/admission.go

```diff
@@ -1397,10 +1405,24 @@ func isAdmissionRuleMet(admResObject *nvsysadmission.AdmResObject, c *nvsysadmis
 			met, positive = isSetCriterionMet(crt, c.ImageRegistry)
 		//case share.CriteriaKeyBaseImage: //-> is base image available?
 		//	met, positive = isStringCriterionMet(crt, scannedImage.BaseName)
+		case share.CriteriaKeyCVECriticalCount:
+			met, positive = isCveCountCriterionMet(crt, false, 0, scannedImage.CriticalVulInfo)
+		case share.CriteriaKeyCVEHighCountNoCritical:
+			met, positive = isCveCountCriterionMet(crt, false, 0, scannedImage.HighVulInfo)
 		case share.CriteriaKeyCVEHighCount:
 			met, positive = isCveCountCriterionMet(crt, false, 0, scannedImage.HighVulInfo)
+			if !met {
```

Comment:
CriteriaKeyCVEHighCount means count of critical+high CVEs
Let's say the criteria is "CriteriaKeyCVEHighCount > 5" and there are 3 critical CVEs and 3 high CVEs found in an image.
In this case condition, 
1. isCveCountCriterionMet(crt, false, 0, scannedImage.HighVulInfo) returns met=false and 
2. isCveCountCriterionMet(crt, false, 0, scannedImage.CriticalVulInfo) returns met=false
But it should get match=true result (3+3 > 5)
Is this logic correct?